spec/seeds_orders_table.sql

```diff
@@ -0,0 +1,15 @@
+-- First, you'd need to truncate the table - this is so our table is emptied between each test run,
+-- so we can start with a fresh state.
+-- (RESTART IDENTITY resets the primary key)
+
+TRUNCATE TABLE orders RESTART IDENTITY; -- replace with your own table name.
```

Comment:
You can put `cascade` at the end of this line to allow the table to be truncated. I guess there is an `on delete cascade` in the foreign key line while creating the table. It prevents you to delete the table as there is a constraint between them.